### PR TITLE
Fix: UK-US location problem

### PR DESCRIPTION
**Pull request by mcamarad**
## Why?

We've discovered a problem with UK/US location being confused due to have the same name. This PR addresses that and situations in which there are ties in order to improve the current system.

## What
- Added edge cases to cover UK/US problem by identifying UK nations and majorities of mentions in the text.
- Updated tests accordingly

### CODE REVIEW

File: geocoder_module/geocoder.py [hunk 1]
```diff
@@ -344,71 +417,96 @@ def double_check_countries(
         # extract the candidate reference countries
         if not top_countries:
             majority = Counter(countries).most_common()
+            ner_majority = Counter(ner_countries_count).most_common()
+            print(ner_majority)
```
Comment (ceru23) [inline geocoder_module/geocoder.py:421]: are these print here intentional?

**Review by ceru23 — COMMENTED** [review]
LGTM! Just a quick comment on some pritns left in the code, if it is intentional consider it as approved

**Review by ceru23 — APPROVED** [review]
Great Job!
